res/values/strings.xml

```diff
@@ -607,6 +607,8 @@
     <string name="pref_send_copy_to_self">Send Copy to Self</string>
     <string name="pref_auto_folder_moves">Automatic Moves to DeltaChat Folder</string>
     <string name="pref_auto_folder_moves_explain">Chat conversations are moved to avoid cluttering the Inbox</string>
+    <string name="pref_only_fetch_mvbox_title">Only fetch from the DeltaChat folder</string>
```

Comment:
```suggestion
    <string name="pref_only_fetch_mvbox_title">Only Fetch from DeltaChat Folder</string>
```

we "headline capitalize" options meanwhile, that means everythings beside to/and/or/from/etc. becomes uppercase